packages/mobile/src/account/Invite.tsx

```diff
@@ -62,12 +71,16 @@ class Invite extends React.Component<Props, State> {
     headerLeft: <CancelButton eventName={CustomEventNames.invite_cancel} />,
   })
 
-  state: State = {
-    searchQuery: '',
-  }
+  state: State = { searchQuery: '', hasGivenPermission: true }
 
   async componentDidMount() {
     this.props.navigation.setParams({ title: this.props.t('invite') })
+
+    PermissionsAndroid.check(PermissionsAndroid.PERMISSIONS.READ_CONTACTS).then(
```

Comment:
Please use utility here for this:
packages/mobile/src/utils/androidPermissions.ts

Reply:
![Screenshot_20190805-153937_Celo (DEV)](https://user-images.githubusercontent.com/1877013/62525941-8e80dd00-b838-11e9-9234-242957ba8f8a.jpg)

Reply:
@jmrossy Attached is the screenshot, and I have updated the permissions code to use the util.